src/vector.lua

```diff
@@ -0,0 +1,29 @@
+local vector = {}
+
+function vector.sign(x)
```

Comment:
I put `sign` in with the vector functions even though it has nothing to do with vectors because it's almost always used by code that also deals with vectors. That's a bad reason, and explaining this makes me want to change it.

Reply:
One function is not enough to make a different module. Change it in a different branch when we know what kind of stuff goes with it.